plugins/woocommerce-admin/client/launch-your-store/hub/main-content/pages/launch-store-success/Congrats.tsx

```diff
@@ -0,0 +1,277 @@
+/* eslint-disable @woocommerce/dependency-group */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { getSetting } from '@woocommerce/settings';
+import { recordEvent } from '@woocommerce/tracks';
+import { createInterpolateElement, useState } from '@wordpress/element';
+import { Link, ConfettiAnimation } from '@woocommerce/components';
+import { isInteger } from 'lodash';
```

Comment:
There's a desire to remove dependency from lodash where possible. Probably best not to add more, see pdToLP-yS-p2